test/perl/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+set(PERL_FILES AccessNode.pm DataNode.pm TimescaleNode.pm)
+
+foreach(P_FILE ${PERL_FILES})
+  configure_file(${P_FILE} ${CMAKE_CURRENT_BINARY_DIR}/${P_FILE} COPYONLY)
```

Comment:
Why do we need to copy the files to the binary (build) dir?

Reply:
Removed. This is indeed unwanted.